code/modules/research/designs/bluespace_designs.dm

```diff
@@ -5,7 +5,7 @@
 	name = "Artificial Bluespace Crystal"
 	desc = "A small blue crystal with mystical properties."
 	id = "bluespace_crystal"
-	req_tech = list("bluespace" = 4, "materials" = 6)
+	req_tech = list("bluespace" = 2)
```

Comment:
This was likely high on purpose, otherwise you can easily max telescience. The item had barely any tech levels so RnD wouldn't quickly steal these from telescience and get tons of free research.